content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+       image: nginx:1.14.2
+```
+ 
+Tokens de Contas de serviço são perfeitamente válidos para ser usados fora do cluster e podem ser utilizados para criar identidades para processos de longa duração que desejem comunicar-se com a API do Kubernetes. Para criar manualmente uma conta de serviço, utilize-se simplesmente o comando `kubectl create serviceaccount (NOME)`. Isso cria uma conta de serviço e um segredo associado a ela no namespace atual.
+ 
+```bash
+kubectl create serviceaccount jenkins
+```
+ 
+```none
+serviceaccount "jenkins" created
+```
+ 
+Verificando um segredo associado:
+ 
+```bash
+kubectl get serviceaccounts jenkins -o yaml
+```
+ 
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+ # ...
+secrets:
+- name: jenkins-token-1yvwg
+```
+O segredo criado irá armazenar a autoridade de certificado do servidor de API e um JSON Web Token (JWT) digitalmente assinado.
+ 
+```bash
+kubectl get secret jenkins-token-1yvwg -o yaml
+```
+ 
+```yaml
+apiVersion: v1
+data:
+ ca.crt: (APISERVER'S CA BASE64 ENCODED)
+ namespace: ZGVmYXVsdA==
+ token: (BEARER TOKEN BASE64 ENCODED)
+kind: Secret
+metadata:
+ # ...
+type: kubernetes.io/service-account-token
+```
+ 
+{{< note >}}
+Valores são codificados em base64 porque segredos são sempre codificados neste formato.
+{{< /note >}}
+ 
+O JWT assinado pode ser usado como um _bearer token_ para autenticar-se como a conta de serviço. Veja [acima](#Colocando um _bearer token_  em uma requisição) para como o _token_ pode ser incluído em uma requisição. Normalmente esses segredos são montados no pod para um acesso interno ao cluster ao servidor de API, porém pode ser utilizado fora do cluster também.
+ 
+Contas de serviço são autenticadas com o nome de usuário `system:serviceaccount:(NAMESPACE):(SERVICEACCOUNT)` e são atribuídas aos grupos `system:serviceaccounts` e `system:serviceaccounts:(NAMESPACE)`.
+ 
+AVISO: porque os _tokens_ das contas de serviço são armazenados em segredos, qualquer usuário com acesso de leitura a esses segredos podem autenticar-se como a conta de serviço. Tome cuidado quando conceder permissões a contas de serviços e capacidade de leitura de segredos.
+ 
+### Tokens OpenID Connect
+ 
+[OpenID Connect](https://openid.net/connect/) é uma variação do framework de autorização OAuth2 que suporta provedores como Azure Active Directory, Salesforce, e Google. A principal extensão do OAuth2 é um campo adicional de _token_ de acesso chamado [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken). Este _token_ é um tipo de JSON Web Token (JWT) com campos bem definidos, como usuário, e-mail e é assinado pelo servidor de autorização.
+ 
+Para identificar o usuário, o autenticador usa o `id_token` (e não `access_token`) do _bearer token_ da resposta de autorização do  OAuth2 [token response](https://openid.net/specs/openid-connect-core-1_0.html#TokenResponse). Veja [acima](#Colocando um _bearer token_  em uma requisição) como incluir um _token_ em uma requisição.
```

Comment:
The same case https://github.com/kubernetes/website/pull/26928/files#r589054963

Reply:
Resolved at 9df194e